### PR TITLE
Add missing enum values in S.Reflection.Emit

### LINKED ISSUE #13486: 3 enums missing entries in Ref.Emit
These are in NS2.0 but missing from our copies of the enums:

```
namespace System.Reflection.Emit {
    public enum FlowControl {
        Phi = 6,
    }
    public enum OpCodeType {
        Annotation = 0,
    }
    public enum OperandType {
        InlinePhi = 6,
    }
```

If they don't have meaning in Core, they should be present but using them can throw PNSE.

### PR DESCRIPTION
**Pull request by alexperovich**
Fixes #13486

### CODE REVIEW

File: src/System.Reflection.Primitives/ref/System.Reflection.Primitives.cs [hunk 1]
```diff
@@ -294,6 +296,7 @@ public enum OperandType
         InlineI8 = 3,
         InlineMethod = 4,
         InlineNone = 5,
+        InlinePhi = 6,
```
Comment (danmosemsft) [inline src/System.Reflection.Primitives/ref/System.Reflection.Primitives.cs]: Obsolete attributes need to be in the ref, if they were in standard. Eg in standard I see

```
        [System.ObsoleteAttribute("This API has been deprecated. http://go.microsoft.com/fwlink/?linkid=14202")]
        InlinePhi = 6,
```

Comment (danmosemsft) [inline src/System.Reflection.Primitives/ref/System.Reflection.Primitives.cs]: all three

**Comment by justinvp** [comment]
Should the obsolete attributes be added to the implementation as well for readability?

**Comment by justinvp** [comment]
I went ahead and submitted #13763

**Pull request merged**